tasks/kmt.py

```diff
@@ -972,12 +972,13 @@ def kmt_sysprobe_prepare(
                 "fmapper",
                 "prefetch_file",
                 "fake_server",
+                "sample_service",
             ]:
                 src_file_path = os.path.join(pkg, f"{gobin}.go")
                 if os.path.isdir(pkg) and os.path.isfile(src_file_path):
                     binary_path = os.path.join(target_path, gobin)
                     nw.build(
-                        inputs=[f"{pkg}/{gobin}.go"],
+                        inputs=[f"{pkg}/..."],
```

Comment:
I don't think ninja is going to like this. It wants a list of files.

Reply:
Could you put the other files in a subpackage? Then I think the previous `go build` command line will work just fine.

Reply:
Or put the `main.go` in a subdirectory. See `grpc_external_server` for an example.